src/routes/mods.rs

```diff
@@ -15,13 +16,49 @@ pub async fn mod_search(
     Ok(HttpResponse::Ok().json(results))
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct ModIds {
+    pub ids: String,
+}
+
+#[get("mods")]
+pub async fn mods_get(
+    web::Query(ids): web::Query<ModIds>,
+    pool: web::Data<PgPool>,
+) -> Result<HttpResponse, ApiError> {
+    let mut mods = vec![];
+
+    for id in serde_json::from_str::<Vec<models::ids::ModId>>(&*ids.ids)? {
+        let mod_data = get_mod_from_id(id, &*pool).await?;
+
+        if let Some(data) = mod_data {
+            mods.push(data)
+        }
+    }
```

Comment:
This is inefficient, since it makes a database query for every requested mod.  It appears that it is possible to use `WHERE id IN (...)` using `sqlx`: (https://github.com/launchbadge/sqlx/issues/656#issuecomment-699638262); this would make it more efficient, and make it less likely to break our servers in case someone tries to do what we're doing with curseforge.